android/assets/jsons/Civ V - Gods & Kings/Nations.json

```diff
@@ -105,8 +105,8 @@
 		"startBias": ["Avoid [Jungle]","Avoid [Forest]"],
 		"preferredVictoryType": "Cultural",
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "O great Ramesses, Pharaoh of Egypt, bright as the Sun and mighty as the Nile! Lord of all living things, you lead the greatest of all civilizations, which has gifted the world with writing, advanced mathematics, sculpture, and architecture. The egyptian culture and monuments amaze the world since thousands of years, keeping Egypt to the pinacle of humanity achievement.",
```

Comment:
I disagree with the use of superioty over the rest. Yeah, like I say that they should be more modest.

Reply:
That is canonically how Egypt thought of themselves :)
The last line is missing capitalization for Egyptian, and needs general rewording:
"Egyptian monuments and culture will continue to amaze the world for thousands of years" without the end bit